.eslintrc

```diff
@@ -29,6 +29,16 @@
         "argsIgnorePattern": "^_"
       }
     ],
+    "header/header": [
+      2,
+      "block",
+      [
+        " This Source Code Form is subject to the terms of the Mozilla Public",
+        " * License, v. 2.0. If a copy of the MPL was not distributed with this",
+        " * file, You can obtain one at https://mozilla.org/MPL/2.0/. "
```

Comment:
nitpick: Super nit, but I think we could shorten this to "SPDX-License-Identifier: MPL-2.0".

Reply:
I think this is a good idea, but for consistency's sake I'd want to do it in the main ironfish repository to start with. Searching around a bit, it might be a good idea to add a copyright line as well